packages/core/src/utils/prepareEvent.ts

```diff
@@ -112,6 +113,54 @@ function applyClientOptions(event: Event, options: ClientOptions): void {
   }
 }
 
+/**
+ * Applies debug metadata images to the event in order to apply source maps by looking up their debug ID.
+ */
+export function applyDebugMetadata(event: Event, stackParser: StackParser): void {
+  const debugIdMap = GLOBAL_OBJ._sentryDebugIds;
+  if (debugIdMap) {
```

Comment:
I would invert this, to do an early return:

```js
if (!debugIdMap) {
  return;
}
```

This is more readable IMHO.

Reply:
Good point. Never was a big early return person but here it makes sense.